customer/urls.py

```diff
@@ -3,5 +3,8 @@
 
 
 urlpatterns = [
-    path("customerdashboard/", views.customerdashboard, name= "customerdashboard")
+    path("customerdashboard/", views.customerdashboard, name= "customerdashboard"),
```

Comment:
nit: customerdashboard could also be written as customer_dashboard for better readability